module/skill/GetUp/data/config/GetUp.yaml

```diff
@@ -0,0 +1,22 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+
+scripts:
+  getup_upright:
+   - "Stand.yaml"
+  getup_front:
+   - "GetUpFront.yaml"
+   - "Stand.yaml"
+  getup_back:
+   - "GetUpBack.yaml"
```

Comment:
we can't get up from the back

Reply:
Should change this config to do what you need to then :P